README.md

```diff
@@ -1,7 +1,190 @@
-### flux-multi-cluster-utilities
-Repository for Fractale's scheduling thrust artifacts.
+## Flux Multi-Cluster Utilities
+This repository houses the artifacts developed as part of the 
+FRACTALE SI's Center-Level Scheduling Thrust. 
+One of the goals of this thrust is to enable scheduling jobs across different 
+clusters of a site, with the objective of 
+improving job performance, turnaround times, system throughput and utilization. 
+A future goal is to enable such scheduling across both HPC clusters and on-premises 
+Cloud setups.
 
-### License
+To this effect, the first building block is a jobtap plugin 
+that allows a job to be delegated (submitted) from the 
+current flux instance to a different flux instance (given its URI).
+
+### Build Instructions
+
+We use GNU Autotools to build this plugin as follows.
+
+```
+./autogen.sh
+./configure --prefix=<install-path>
+make && make install
+```
+The `delegate.so` plugin will be installed in `<install-path>/lib/flux/job-manager/plugins/`.
+
+If no `<install-path>` is specified, it is installed in `flux-multi-cluster-utilities/src/job-manager/plugins/.libs`.
+
+### Loading a JobTap Plugin
+The plugin can be loaded with the command:
+`flux jobtap load <path-to-plugin>/delegate.so`
+
+### Interactive Testing on Peer-to-Peer Flux Instances
+
+Here, we show an example of peer-to-peer flux instances, residing on the same cluster
+and belonging to the same user. Enabling testing across flux instances on different clusters
+is ongoing research. 
+
+This plugin was tested on the Corona cluster across a 4 node allocation.
+Similar steps can be performed on any other cluster. 
+
+#### 1. Obtain an interactive allocation on the desired cluster.
+```
+flux alloc -N4      # Default resource manager is Flux, e.g. Corona
+salloc -N4          # Default resource manager is SLURM
+```
+_Note_: If the default resource manager is SLURM, a `flux start -N4` will be needed to start a top-level instance.
```

Comment:
On slurm it would have to be `srun -N4 -n4 flux start -N4` or `srun --tasks-per-node=1 flux start`